Makefile

```diff
@@ -49,7 +58,7 @@ update-assets:
 # Once we change CI code to build outside GOPATH, GO111MODULE can be removed, so
 # we rely on defaults.
 build-slim:
-	CGO_ENABLED=0 GOOS=linux GO111MODULE=on go build \
+	CGO_ENABLED=0 GOOS=$(GOOS) GO111MODULE=on go build \
```

Comment:
Hm, why do we need to set GOOS explicitly? Can we just remove it?

Reply:
The produced binaries are exactly the same. If building on MacOS works just fine, I guess we can just leave it as it is?

Reply:
Yes, let me update the PR.

Reply:
I'll actually test that on macOS before we remove it to understand how things work.

Reply:
I have macOS, I tested it, you can double check

Reply:
If we build without it, it will work locally. But I guess for release builds, if we want to provide binaries for both macOS and Linux, we need to specify it then, to do cross-compile, right?

Reply:
Just checked, we already do via goreleaser. All good, cool!